ontology/yaml/resources/units/units.yaml

```diff
@@ -192,6 +192,8 @@ powerfactor:
 powerflux:
 - watts_per_square_meter: STANDARD
 - watts_per_square_foot
+precipitation:
```

Comment:
looks like a distance, or a level, which are lengths. this seems more like a descriptor.